### PR TITLE
Port materialize css to 1.0.0

### PR DESCRIPTION
**Pull request by jenow**
## What does this PR do ?

Port materialize CSS to version 1.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-admin-console",
-  "version": "2.2.4",
+  "version": "2.3.4",
```
Comment (scottinet) [inline package.json]: why do you change the version number manually to release on the development branch?

Comment (scottinet) [reply package.json]: Also, if you want the minor version to be updated, then you should use the `changelog:new-features": this triggers a minor version update from our release tools.

Comment (jenow) [reply package.json]: I don't want to update the last number as we change it for every hotfix, I want to change the second one. It should be 2.3.0 btw -.-

Comment (scottinet) [reply package.json]: You don't get it: if you manually update the version and release the admin console with our tools, you'll get a "2.3.1" version tag (unless you use `--tag` to force the future version number to 2.3.0).
If you leave the version alone and simply add the `changelog:new-features` label, then the release tools will automatically increment the minor version number.

Comment (jenow) [reply package.json]: Right, I'm sorry I did some hotfix before and I was used to change the version, forgetting I was making a PR on the development branch -_-

**Pull request merged**